sample/src/main/java/com/springboot/hello/sample/model/admin.java

```diff
@@ -0,0 +1,52 @@
+package com.springboot.hello.sample.model;
+
+
+public class admin {
```

Comment:
Is this you? Writing a java class without following naming conventions? 🤦  When it comes to Spring and Spring boot, it is very very important to follow conventions strictly. BTW this class is not needed since the requirement was just to have a single resource that takes username in `query param` and return `Hello <User Name>`